src/App.js

```diff
@@ -1,23 +1,43 @@
-import logo from './logo.svg';
 import './App.css';
+import Counter from "./components/counter/Counter";
+import { SearchForm } from "./components/search-form/SearchForm";
+import "@fontsource/montserrat";
+import "@fontsource/montserrat/400.css";
+import "@fontsource/montserrat/400-italic.css";
+import { GenreSelect } from "./components/genre-select/GenreSelect";
+import React from "react";
+
+export const listGenres = [
+    {
+    id: '1', name: 'Crime'
+    }, {
+    id: '2', name: 'Documentary'
+    }, {
+    id: '3', name: 'Horror'
+    }, {
+    id: '4', name: 'Comedy'
+    }
+]
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <h2 style={{textAlign: 'left'}}>Epam React Course</h2>
+        <hr/>
+        <div>
+            <header style={{textAlign: 'center'}}>Counter component</header>
```

Comment:
The Inline CSS is not a good approach for this course, Please try to use the CSS File to declare the App Styles